server/pbench/bin/pbench-verify-s3-backup-tarballs

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+import glob
+import hashlib
+import boto3
+import configtools
+
+options = []
+opts, args = configtools.parse_args(options)
+conf, files = configtools.init(opts)
+
+# configuration file
+endpoint_url = conf.get('aws-s3', 'endpoint_url')
+aws_access_key_id = conf.get('aws-s3', 'aws_access_key_id')
+aws_secret_access_key = conf.get('aws-s3', 'aws_secret_access_key')
+bucket_name = conf.get('aws-s3', 'bucket_name')
+target_dir = conf.get('aws-s3', 'target_dir')
+
+s3 = boto3.resource('s3', aws_access_key_id='%s' % aws_access_key_id,
```

Comment:
Can we delay the S3 object setup until we are ready to fetch the bucket list?

That way, if we can't fetch the S3 data, we can still verify our local copy and our backup copy.

Reply:
Done